api/src/main/java/cz/muni/fi/pa165/sportsclub/facade/PlayerFacade.java

```diff
@@ -0,0 +1,21 @@
+package cz.muni.fi.pa165.sportsclub.facade;
+
+import cz.muni.fi.pa165.sportsclub.dto.PlayerDto;
+
+/**
+ * Facade for player
+ *
+ * @author 422636 Adam Krajcik
+ */
+public interface PlayerFacade {
+
+    void createPlayer(PlayerDto player);
+
+    void updatePlayer(PlayerDto player);
+
+    void deletePlayer(PlayerDto player);
+
+    void getPlayer(Long id);
```

Comment:
In my opinion, get player by bussines key (email, if I remember correctly) should be possible too (same for coach)